src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -244,6 +270,12 @@ const MeasureGroups = () => {
   };
 
   const submitForm = (group: Group) => {
+    if (group.stratifications) {
+      group.stratifications = group.stratifications.filter(
+        (strat) => !!strat.description || !!strat.cqlDefinition
```

Comment:
what is the benefit of using double negation here?

Reply:
In typescript !! means truthiness of the object.